### ISSUE TITLE
TypeOperator not supported by ts-interface-builder: keyof SomeInterface

### ISSUE THREAD

**Issue by TomascpMarques** [gristlabs/ts-interface-checker#54]
## TL;DR
Be able to use `field: keyof SomeInterface` as an interfaces field type, without needing to export custom types.

## What's the issue?
Using `keyof SomeInterface`, as an interface field, type__throws out an error__ :
` Node TypeOperator not supported by ts-interface-builder: keyof IUser`

## What happened?
So I was defining a new interface, and in one of the fields I defined the as `field: keyof IUser`, and when I ran the:
`npx ts-interface-builder foo.ts` it gave me the previously mentioned error.

## What did you try to fix the issue?
After the error message, I used the [Type suites](https://github.com/gristlabs/ts-interface-checker#type-suites), exported the type, rewrote the previous error producing code using the exported type, and it works.

## What do you want?
Be able to use `field: keyof OtherInterface` directly, instead of exporting a type, pretty pls 😇

### P.S 
> I'm loving your package, eases my development experience tremendously, loving it <3
